pkg/features/kube_features.go

```diff
@@ -904,7 +904,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	HugePageStorageMediumSize:                      {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.24
 	DownwardAPIHugePages:                           {Default: true, PreRelease: featuregate.Beta},                    // on by default in 1.22
 	AnyVolumeDataSource:                            {Default: false, PreRelease: featuregate.Alpha},
-	DefaultPodTopologySpread:                       {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Since PodTopologySpread graduated to GA, should we remove SelectorSpread then? @alculquicondor

Reply:
We can't, because it's a supported plugin in v1beta3. We can potentially remove it once we release a v1. Maybe we will do that in 1.25.

Reply:
The feature gate itself should be removed two releases after graduation. That's 1.26.

Reply:
Or we can add some related comments. Forget it if not necessary.